Source/SizableCollectionCollectionDataDisplayManager/SizableCollectionCollectionDataDisplayManager.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  SizableCollectionCollectionDataDisplayManager.swift
+//  ReactiveDataDisplayManager
+//
+//  Created by Alexander Filimonov on 02/03/2020.
+//  Copyright © 2020 Александр Кравченков. All rights reserved.
+//
+
+import Foundation
+
+public class SizableCollectionCollectionDataDisplayManager: BaseCollectionDataDisplayManager {
```

Comment:
SizableCollectionDataDisplayManager is that what u wanted?) or really  SizableCollectionCollectionDataDisplayManager?

Reply:
yes, just mistake